pkgs/os-specific/linux/kernel/manual-config.nix

```diff
@@ -185,12 +198,14 @@ let
       '';
 
       buildFlags = [
+        "DTC_FLAGS=-@"
         "KBUILD_BUILD_VERSION=1-NixOS"
-        kernelConf.target
-        "vmlinux"  # for "perf" and things like that
-      ] ++ optional isModular "modules"
```

Comment:
This PR inadvertently destroyed the ability to build a kernel without the modules, which is a *much* faster build.

Reply:
https://github.com/NixOS/nixpkgs/pull/244447